pkg/agent/proxy/proxier.go

```diff
@@ -470,25 +482,33 @@ func (p *proxier) installServices() {
 				klog.ErrorS(err, "Error when installing Endpoints flows")
 				continue
 			}
+			var exportedSvcGroupID binding.GroupIDType
+			if p.multiclusterEnabled && svcPortName.Name != mcWebhookServiceName && strings.HasPrefix(svcPortName.Name, "antrea-mc-") {
+				exportedSvcPortName := svcPortName
```

Comment:
Maybe add some comments here since for developers that are not familiar with mc-cluster don't know why the Service name is got by removing prefix `antrea-mc-`.

Reply:
Maybe add some comments to explain why excluding Service `mcWebhookServiceName`?